content/manual/running/vere.md

```diff
@@ -290,14 +314,24 @@ previously with `cram` or `-n`.
 Download a binary. `DIR` is an output directory (it must already exist) and `ARGS` are:
 
 - `-a, --arch ARCH` - architecture, `ARCH` may be one of `x86_64-linux`,
-  `x86_64-darwin`, and `x86_64-windows`, though more may be added in the future.
+  `x86_64-darwin`, `x86_64-windows` and `aarch64-linux`, though more may be
+  added in the future.
 - `-v, --version VER` - version number, `VER` is e.g. `1.9`.
 - `-p, --pace` - release channel, e.g. `live`.
 
 Example usage:
 
 - Undocked: `urbit vere -a x86_64-linux -v 1.9 -p live .`
 
+### `vile`
+
+Export keyfile.
```

Comment:
What's the expected utility of `vile`?

Reply:
idk tbh, I can't think of a non-obscure reason to use it